barretenberg/acir_tests/sol-test/src/index.js

```diff
@@ -179,8 +208,20 @@ try {
   const proofPath = getEnvVar("PROOF");
   const proof = readFileSync(proofPath);
 
-  // Cut the number of public inputs off of the proof string
-  const proofStr = `0x${proof.toString("hex").substring(64 * numPublicInputs)}`;
+  // Cut the number of public inputs out of the proof string
+  let proofStr = proof.toString("hex");
+  if (testingHonk) {
```

Comment:
The verifier's interface is verify(proof, publicInputs[]). However, in it's current form honk proofs have 3 elements before the public inputs, so I (and the user) will need to slice these out in order to have proofs which can be processed by the verifier. 

I did not want to change the serialization of the proofs in this pr, but it is something to think about ahead of the noir switch over